### PR TITLE
First pull request

### PR DESCRIPTION
**Pull request by mricanho**
In this pull request, I finish the project with a beautiful "Hello Microverse"

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.first {
+  margin: auto;
+  width: 50%;
+  border: 3px solid green;
```
Comment (Shaher-11) [inline style.css]: Please make sure to align the border as it's not aligned properly now

File: style.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  padding: 10px;
+}
+
+h1 {
+  color: rgba(13, 13, 117, 0.623);
+  margin: auto;
+  width: 50%;
```
Comment (Shaher-11) [inline style.css]: You need to use `display:inline-block` if you wish to specify the width of h1 tag

**Review by Shaher-11 — COMMENTED** [review]
Hello @mricanho 

#### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:blush::blush:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [x] You need to use display:inline-block as the width will not apply on the h1 tag
- [x] aligning the border is not working as expected

## Optional Suggestions 💡
_The following post are highly recommend to improve your code quality, Please consider implementing them

- The commits have to be more descriptive so instead of `Add files` maybe `Add section and style section head`

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @mricanho 

#### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:blush::blush:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [x] You need to use display:inline-block as the width will not apply on the h1 tag
- [x] aligning the border is not working as expected

## Optional Suggestions 💡
_The following post are highly recommend to improve your code quality, Please consider implementing them

- The commits have to be more descriptive so instead of `Add files` maybe `Add section and style section head`

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Review by jubaan — APPROVED** [review]
Hi @mricanho , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**